### DISCUSSION TITLE
Installing recharts gives you an old version 0.2.16

### DISCUSSION THREAD

**Discussion by vitalijalbu** [recharts/recharts#3092] [General]
Hi guys, just noticed that I have a 0.2.16. version (installed yesterday), but if running npm outdated, it give you the 2.2.0. How come??

**Comment by ckifer** [comment]
Hi @vitalijalbu this might be because there hasn't been tags for the past few versions. Not sure why this is but I would attribute it to that.

**Reply by ckifer** [reply]
@vitalijalbu tags should be added up to the latest version now. This should be resolved.
